app/uk/gov/hmrc/agentclientrelationships/services/DeleteRelationshipsService.scala

```diff
@@ -138,15 +138,19 @@ private[services] abstract class DeleteRelationshipsService(
                          enrolmentKey,
                          arn
                        ) // TODO DG oneTaxIdentifier may not return what we want for CBC!
-      if maybeResponse.nonEmpty
-      _ = auditData.set(etmpRelationshipRemovedKey, true)
+      _ = if (maybeResponse.nonEmpty) auditData.set(etmpRelationshipRemovedKey, true) else ()
```

Comment:
if block that returns a Unit does not need an explicit else case

Reply:
Changed